src/go/MONIT/ggus_parser.go

```diff
@@ -291,25 +298,88 @@ func processResponse(url, format, accept, out string) {
 		}
 	}
 
+	return resp
+}
+
+func retrieveTicketDetails(ticketId int) (string, []TicketComment) {
+	url := fmt.Sprintf("https://ggus.eu/?mode=ticket_info&ticket_id=%d", ticketId)
+	resp := ggusRequest(url, "text/html")
+	fmt.Println(resp)
+
+	doc, _ := html.Parse(resp.Body)
+
+	description := ""
+	var comments []string
+
+	var f func(*html.Node, bool, bool)
+	f = func(n *html.Node, isDescription bool, isHistory bool) {
```

Comment:
please add proper comment why you used function in place of another function. Reading the code I see it is recursive but it may not be obvious in a first place.